types/luxon/src/duration.d.ts

```diff
@@ -1,5 +1,6 @@
 import { NumberingSystem } from './misc';
 import { ConversionAccuracy } from './datetime';
+import { IfInvalid } from './_util';
```

Comment:
does it make sense to use 'impl/util.js' setup here?

```ts
import type { IfInvalid } from './impl/util'
```
the util file would export types only here (without anything else from a module), so internal content, no re-exported/exposed.

Reply:
Tomato tomato to me. Neither are exported via index so they can only be accessed directly
```ts
import type { IfInvalid } from 'luxon/_util';
import type { IfInvalid } from 'luxon/impl/util';
```
I can move if you think that would be better.

Reply:
Ah I gotcha. I could go either way. On the other hand this is logic that only exists here not in the actual src code. And it's not like we'd have any other implementation code here right?